lib/streamaggr/streamaggr.go

```diff
@@ -111,6 +111,13 @@ type Options struct {
 	//
 	// This option can be overridden individually per each aggregation via ignore_old_samples option.
 	IgnoreOldSamples bool
+
+	// IgnoreFirstIntervals sets amount of intervals to ignore on start
+	//
+	// By default no intervals will be ignored.
+	//
+	// This option can be overridden individually per each aggregation via ignore_intervals_on_start option.
```

Comment:
```suggestion
	// This option can be overridden individually per each aggregation via ignore_first_intervals option.
```

Reply:
Although, I don't see `ignore_first_intervals` setting to have an actual effect. Looks like current PR respects only flag value